tests/src/cgeo/geocaching/location/GeoPointParserTest.java

```diff
@@ -277,7 +277,7 @@ public void test5790() {
         GeopointParser.parse("N52 36.123 E010 06.456");
         GeopointParser.parse("52° 10°");
         GeopointParser.parse("52° -10°");
-        GeopointParser.parse("52.55123 10,56789");
+        GeopointParser.parse("52,55123 10,56789");
         GeopointParser.parse("52.55123° 10.56789°");
```

Comment:
We need also a test for something with "," between N and E part, like
`Nxx° yy.zzz', Exx° yy.zzz'`

Reply:
That's covered by test8845 added below.

Reply:
oh, yes.